src/test/java/step2/StringUtilTest.java

```diff
@@ -0,0 +1,33 @@
+package step2;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
+
+public class StringUtilTest {
+
+    @DisplayName("입력 문자열이 공백값을 기준으로 배열로 변환된다")
+    @Test
+    public void splitBySpace_ShouldReturnArray() {
+        String inputString = "10 + 15 - 7 * 3 / 9";
+
+        StringUtil stringUtil = new StringUtil(inputString);
```

Comment:
```suggestion
        StringCalculator stringCalculator = new StringCalculator("10 + 15 - 7 * 3 ^ 9");
```
불필요한 로컬변수는 inline 하면 가독성이 좋아져요.
로컬변수가 존재하면 컴퓨터는 메모리에 저장하고 사람은 기억해야 하는데 사람의 경우엔 임시로 기억해야하는 대상이 많아질수록 기독성이 떨어진다고 느끼거든요.

Reply:
감사합니다. 또 하나 배우네요😃